samples/templates/default-azuredeploy-docker.json

```diff
@@ -487,6 +487,19 @@
                 "[resourceId('Microsoft.Sql/servers', variables('serviceName'))]"
             ]
         },
+        {
```

Comment:
Could this value be calculated in the run-test file instead of being added to the deployment template? The templates are used by customers to deploy their systems and I don't think we should have test code in here.

Reply:
Agree. Since the acr account names are generated with [uniqueString](https://github.com/microsoft/fhir-server/blob/1a0c0bffdc75f376e96c5e3a19b109ea23f97ad3/samples/templates/default-azuredeploy-docker.json#L235) method, we can only read the value from service configuration where we register the ACR login servers. I have removed this from deployment template